packages/graph/.env

```diff
@@ -1,17 +1,11 @@
-HASH_GRAPH_PG_USER=postgres
-HASH_GRAPH_PG_PASSWORD=postgres
-HASH_GRAPH_PG_HOST=localhost
-HASH_GRAPH_PG_PORT=5432
```

Comment:
I've removed these in favour of `POSTGRES_PORT` and `POSTGRES_HOST` because the host and port are the same for all databases in the postgres instance